### PR TITLE
Added Apache v2 LICENSE

### PR DESCRIPTION
**Pull request by malex984**
1. Added LICENSE
2. Added NOTICE
3. Added references to README.md

### CODE REVIEW

**Comment by malex984** [comment]
Please check the added NOTICE file.

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,15 @@
+-  Hilbert Docker Images
+
+   Copyright 2016 IMAGINARY – open mathematics
```
Comment (porst17) [inline NOTICE]: `IMAGINARY - open mathematics` -> `IMAGINARY gGmbH`

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   Copyright 2016 IMAGINARY – open mathematics
```
Comment (porst17) [inline LICENSE]: `IMAGINARY - open mathematics` -> `IMAGINARY gGmbH`

**Review by porst17 — CHANGES_REQUESTED** [review]
The official name of the company is `IMAGINARY gGmbH`, not `IMAGINARY - open mathematics`.

File: LICENSE (hunk 2 above)
Comment (malex984) [reply LICENSE]: fixed

**Review by porst17 — APPROVED** [review]

**Pull request merged**